ssc/cmod_pvsamv1.cpp

```diff
@@ -2793,6 +2810,72 @@ void cm_pvsamv1::exec()
                     ac_subhourlyclipping_loss = 0.0;
                 }
             }
+            if (as_integer("enable_subinterval_distribution") == 1) {
+                if (dcPower_kW > 0.0) {
+                    double acPower_kW_max = dcPower_kW_csky;
+                    double acPower_kW_avg = sharedInverter->powerAC_kW + sharedInverter->powerClipLoss_kW;
+                    acpwr_gross += sharedInverter->powerClipLoss_kW; //add clipping loss back in
+                    sharedInverter->powerClipLoss_kW = 0; //set standard clipping to zero;
+                    //sharedInverter_clipping->calculateACPower(dcPower_kW_csky, dcVoltagePerMppt[0], Irradiance->weatherRecord.tdry);
+                    acPower_kW_max = std::max(PVSystem->p_systemACPowerMax[idx], acPower_kW_avg);
+                    log(util::format("dcPower max is %lg", acPower_kW_max), SSC_NOTICE);
+                    double alt_angle = Irradiance->p_sunAltitudeAngle[idx] * M_PI / 180;
+                    if (Irradiance->p_sunAltitudeAngle[idx] < 0.001) alt_angle = 0.001 * M_PI / 180; //For negative alt angle
+                    double AM = 1.0 / sin(alt_angle);
+                    if (AM > 38.0) AM = 38.0;
+                    double acPower_kW_min = std::min(acPower_kW_max * 0.045 / AM, acPower_kW_avg); //AM?
+                    log(util::format("dcPower min is %lg", acPower_kW_min), SSC_NOTICE);
```

Comment:
Looks like there are some log notices throughout this function that need to be removed

Reply:
Deleted.